src/schema.graphql

```diff
@@ -181,6 +193,19 @@ type RequestedChange {
     team: Team
 }
 
+input RequestedChangeInput {
+    gcID: ID!
+    name: String
+    email: String
+    avatar: String
+    mobilePhone: PhoneNumber
+    officePhone: PhoneNumber
+    address: AddressInput
+    titleEn: String
+    titleFr: String
+    team: ModifyTeamInput
```

Comment:
ModifyTeamInput should be changed to TeamInput.  We only need the id of the team to link to.